src/cpp/src/sequence_group.hpp

```diff
@@ -446,6 +446,10 @@ class SequenceGroup {
         return m_generation_stream->get_status() == GenerationStatus::DROPPED_BY_HANDLE;
     }
 
+    void push_empty() {
```

Comment:
```suggestion
    void push_empty_outputs() {
```

Please also add a comment why we need that method.

Reply:
Please check, there already is a comment


            // Push anything to the queue to unblock cancelled read_all() calls
            // When handle is dropped we do not care about any remaining data